WcaOnRails/spec/requests/api_competitions_spec.rb

```diff
@@ -18,30 +76,13 @@
         end
       end
 
-      context "when signed in as an admin" do
-        sign_in { FactoryBot.create :admin }
+      context "when signed in as a board member" do
+        sign_in { FactoryBot.create :board_member }
 
+        #Test might be redundant?
```

Comment:
What is this redundant with? It looks like this is unique because it's a test of editing an unconfirmed competition.

Reply:
My thinking was that there wouldn't be an instance where a board member would be able to modify the events of a confirmed competition but not an unconfirmed competition. For now, I will keep the test but clarify the name of the test with the words "of an unconfirmed competition."